project.toml

```diff
@@ -0,0 +1,13 @@
+name = "MusicProcessing"
+uuid = "   "
```

Comment:
But we need a `uuid` though :) 

How did you generate this file?

Reply:
I called that Pkg.generate("MusicProcessing") 
I am doubtful if the name is "MIDI and repo is  "https://github.com/JuliaMusic/MIDI.jl.git"
```jl
Generating  project MusicProcessing:
    MusicProcessing\Project.toml
    MusicProcessing\src/MusicProcessing.jl
Dict{String,Base.UUID} with 1 entry:
  "MusicProcessing" => UUID("1928ec60-b80c-4700-9146-bdd258a6d4ff")

julia> name = "MIDI"
"MIDI"

julia> uuid = "f57c4921-e30c-5f49-b073-3f2f2ada663e"
"f57c4921-e30c-5f49-b073-3f2f2ada663e"

julia> repo = "https://github.com/JuliaMusic/MIDI.jl.git"
"https://github.com/JuliaMusic/MIDI.jl.git"

julia> version = "1.8.1"
"1.8.1"

julia>

julia> [compat]
ERROR: UndefVarError: compat not defined
Stacktrace:
 [1] top-level scope at REPL[36]:1

julia> julia = "1"
"1"

julia>

julia> [extras]
ERROR: UndefVarError: extras not defined
Stacktrace:
 [1] top-level scope at REPL[38]:1

julia> Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
"8dfed614-e22c-5e08-85e1-65c5234f0b40"

julia>

julia> [targets]
ERROR: UndefVarError: targets not defined
Stacktrace:
 [1] top-level scope at REPL[40]:1

julia> test = ["Test"]
```

Reply:
Thank you!! :)